tools/c7n_azure/c7n_azure/provisioning/deployment_unit.py

```diff
@@ -9,12 +9,12 @@
 
 @six.add_metaclass(ABCMeta)
 class DeploymentUnit:
+    log = logging.getLogger('custodian.azure.deployment_unit')

```

Comment:
`DeploymentUnit`?

Reply:
Yeah see this is where I internally debated module vs class. The file name is `deployment_unit` and there are no other classes in here. For consistency sake if I keep it in the class I'll ad the class name too.